Neo4j.Driver/Neo4j.Driver/Auth/IAuthToken.cs

```diff
@@ -15,7 +15,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-namespace Neo4j.Driver;
+// ReSharper disable once CheckNamespace - public type moved into Auth foldernamespace Neo4j.Driver;
```

Comment:
public API , shouldn't move this to a different namespace, that'd be a breaking change.

Reply:
Sorted